src/main/bin/ccm

```diff
@@ -10,6 +10,7 @@ use warnings;
 # required for CAF
 BEGIN {
     unshift(@INC, '/usr/lib/perl');
+    pop @INC if $INC[-1] eq '.';
```

Comment:
title should be changed to "only remove '.' when it's the last element in INC".
or replace the BEGIN block with (untested)
```perl
BEGIN {
    @INC = ('/usr/lib/perl', grep {$_ ne '.'} @INC);
}
```
btw, this should be turned in a template in maven-tools, i made https://github.com/quattor/maven-tools/issues/169

Reply:
The idea comes from here (a respected member of the perl community):
https://www.masteringperl.org/2017/01/perl-v5-26-removes-from-inc-but-dont-think-youre-safe/

dot is always last in INC so it should suffice.

Reply:
ok, i wasn't sure. can you also open an issue to switch remaining scripts to taint mode, so this isn't forgotten?

Reply:
done